airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml

```diff
@@ -20,6 +20,11 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: Brand Analytics Reports will be removed from Cloud verison. Deprecated FBA reports will be removed permanently.
```

Comment:
can we talk about why/what the benefit is?

Reply:
Deprecated FBA reports will be removed permanently from Cloud and Brand Analytics Reports will be removed temporarily.  Updates on Brand Analytics Reports can be tracked here: https://github.com/airbytehq/airbyte/issues/32353